roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -75,10 +75,10 @@ ntp_enabled: false
 # The default is ntp for most platforms, or chrony on RHEL/CentOS 7 and later.
 # The ntp_package can be one of ['ntp','chrony']
 ntp_package: >-
-      {% if ansible_os_family == "RedHat" -%}
-      chrony
-      {%- else -%}
+      {%- if ansible_os_family == 'Debian' -%}
       ntp
+      {%- else -%}
+      chrony
```

Comment:
Does we need the above change?
The change of roles/kubernetes/preinstall/tasks/0040-set_facts.yml sets "RedHat" for `ansible_os_family`.
and Kubespray will be able to handle those distributions to install chrony even if we don't change here I guess.

Reply:
@oomichi Thanks for your comment, I have updated it according to the relevant suggestions.😁

Reply:
Thanks for updating again!
The new one seems very clean and simple for me :-)